docs/thermometer-skills.md

```diff
@@ -132,6 +132,8 @@ aea config set vendor.fetchai.skills.thermometer.models.strategy.args.currency_i
 aea config set vendor.fetchai.skills.thermometer.models.strategy.args.ledger_id ethereum
 ```
 
+NOTE: If you are not using a Raspberry Pi make sure you make the `has_sensor`  setted to `False`
```

Comment:
`NOTE: If you are not using a Raspberry Pi make sure you set the `has_sensor` attribute to `False`